DasharoModulePkg/Include/Guid/SovereignBoot.h

```diff
@@ -0,0 +1,48 @@
+/** @file
+
+Copyright (c) 2025, 3mdeb Sp. z o.o. All rights reserved.<BR>
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef SOVEREIGN_BOOT_H_
+#define SOVEREIGN_BOOT_H_
+
+#define SOVEREIGN_BOOT_WIZARD_FORMSET_GUID \
+  { \
+    0xB57031B9, 0x1ABB, 0x45F8, {0xA9, 0xCB, 0xAC, 0x5A, 0xAD, 0x72, 0xAD, 0x31} \
+  }
+
+#define SV_BOOT_DATA_VAR      L"SvBootData"
+#define SV_BOOT_CONFIG_VAR    L"SvBootConfig"
+
+// Application launch causes. Determine the logic and screens showed
+// when the application is launched.
+// We want different screens and messages when:
+// 1. SV Boot is proviosioned by image fails to verify.

```

Comment:
Was this the intended wording?
```suggestion
// 1. SV Boot is provisioned but image fails to verify.
```